app/views/payments/_form.html.erb

```diff
@@ -0,0 +1,24 @@
+<%= form_with(model: payment, url: payments_path(def_group: initial_group)) do |f| %>
+  <div class="field">
+    <%= f.label :name %>
+    <%= f.text_field :name, required: true, placeholder: 'Enter name' %>
+  </div>
+  <div class="field">
+    <%= f.label :amount %>
+    <%= f.number_field :amount, required: true, step: "0.01", placeholder: 'Enter amount' %>
+  </div>
```

Comment:
- [ ] Please add a "Categories" field to this form as mentioned in the project requirements.
![image](https://github.com/mohamedSabry0/inqezWallet/assets/45460192/3a2f16ad-8497-405b-8858-afbd52a81b0e)